ansible/roles/prosody/files/mod_websocket_session_ip-0.12.patch

```diff
@@ -0,0 +1,62 @@
+# HG changeset patch
+# User bgrozev
+# Date 1676485553 21600
+#      Wed Feb 15 12:25:53 2023 -0600
+# Branch 0.12
+# Node ID 4480f9f470adb7aff57e65f3c1f11990cd369db1
+# Parent  9ed628635dc66da3fdc28af836efcb1a6428fc95
+Set the session.ip property before the session filters are initialized when using mod_websocket
+
+diff -r 9ed628635dc6 -r 4480f9f470ad core/sessionmanager.lua
+--- a/core/sessionmanager.lua	Thu Feb 09 22:57:32 2023 +0000
++++ b/core/sessionmanager.lua	Wed Feb 15 12:25:53 2023 -0600
+@@ -31,6 +31,7 @@
+ 
+ local function new_session(conn)
+ 	local session = sessionlib.new("c2s");
++    session.ip = conn.http_remote_ip or conn:ip();
```

Comment:
We do not need this one, right?

Reply:
setting the ip is done few lines below in  `sessionlib.set_conn(session, conn);`

Reply:
This is the change we tested on the drained shard

Reply:
Yes, I was porting the previous patch and didn't see it. I'll fix it